test_schemas/en/test_custom_page_titles.json

```diff
@@ -814,11 +814,10 @@
                         },
                         {
                             "type": "Question",
-                            "id": "date-of-birth-block",
-                            "page_title": "Custom question page title",
+                            "id": "date-of-birth",
                             "question": {
                                 "id": "date-pipe-question",
-                                "title": "What is your date of birth?",
+                                "title": "Date of birth",
```

Comment:
I understand this was changed to hide the problem of the question mark in page titles, but this should be reverted - the question title should always be a question. We could add a `page_title` property here as the question text in the page title should be covered by other tests.

Reply:
No problem will revert, but I deliberately left it without a page_title, thus giving me a repeating section with a page_title, but a block without